TOC/README.md

```diff
@@ -0,0 +1,88 @@
+# TiDB Community Technical Oversight Committee (TOC)
+
+The TiDB Community TOC serves as the main bridge and channel for coordinating and information sharing across companies and organizations. It is the coordination center for solving problems in terms of resource mobilization, technical research and development direction in the current community and cooperative projects.
+
+## Responsibilities:
+
+- Partner cooperation project progress monitoring and guidance
+- Supervision and guidance on the progress of the TiDB ecological project
+- Participate in technical route discussion and decision-making, such as version release and roadmap formulation, etc.
```

Comment:
> version release

@winkyao @sunxiaoguang please make sense whether or not it is the case. I don't think TOC will currently or later take responsibility of releases.

Reply:
You are right, I overlooked this rule yesterday. I personally think it is not necessary to plan release in TOC.

Reply:
You are right @tisonkun

Reply:
OK, I will delete the version release.